api/v1/swagger.yaml

```diff
@@ -2205,6 +2205,67 @@ paths:
   # ---------------------------------------------------------------------------
   /collectors:
     x-swagger-router-controller: collectors
+    post:
```

Comment:
I added this back in for now so the heartbeat tests work. Once the auto-registration is set up we can change the tests to use it and remove this again.